reporters/fail.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var PluginError = require('gulp-util').PluginError;
+var through = require('through2');
+
+module.exports = function (opts) {
+	opts = opts || {};
+
+	// paths to files that failed JSCS
+	var fails = false;
+
+	// files that need to be passed downstream on flush
+	var buffer = opts.buffer !== false ? [] : false;
```

Comment:
Well, it is useful in some cases. E.g. if you don't want files to go through to the next stream when one of them contains linting errors, this prevents breaking the pipe during transpiling/testing.

Of course, I guess most people use `gulp-plumber` when they don't want pipes to break unexpectedly, and I believe most people also remove the whole generated directory before starting a new build so having inconsistent state due to a broken pipe isn't such a big deal.

Up to you whether we should it, though.